### ISSUE TITLE
No results in Nextcloud, but Elasticsearch working

### ISSUE THREAD

**Issue by bkanuka** [nextcloud/fulltextsearch_elasticsearch#7]
I have my files indexed an they are searchable with the Elasticsearch API:

For example, if I query Elasticsearch directly for "taxes" I get:
`GET http://192.168.1.10:9200/_search?q=taxes`

``` json
{
    "took": 229,
    "timed_out": false,
    "_shards": {
        "total": 6,
        "successful": 6,
        "skipped": 0,
        "failed": 0
    },
    "hits": {
        "total": 96,
        "max_score": 2.429399,
        "hits": [
            {
                "_index": "nextcloud",
                "_type": "files",
                "_id": "334",
                "_score": 2.429399,
                "_source": {
                    "share_names": [],
                    "owner": "",
                    "users": [],
                    "groups": [],
                    "circles": [],
                    "tags": [
                        "local"
                    ],
                    "title": "redacted"
                }
            },
            {
                "_index": "nextcloud",
                "_type": "files",
                "_id": "361",
                "_score": 2.419319,
                "_source": {
                    "owner": "",
                    "attachment": {
                        "date": "2013-04-27T14:00:02Z",
                        "content_type": "application/pdf",
                        "language": "en",
                        "title": "redacted",
                        "content_length": 73011
                    },
                    "groups": [],
                    "circles": [],
                    "share_names": [],
                    "title": "documents/taxes/redacted.pdf",
                    "users": [],
                    "content": "Executive summary.......
--snipped--
```

This is the Nextcloud log when I search within Nextcloud:
```
Jan 14 09:18:40 server docker[15789]: 127.0.0.1 - - [14/Jan/2018:09:18:40 -0500] "OPTIONS * HTTP/1.0" 200 126 "-" "Apache/2.4.10 (Debian) PHP/7.1.13 (internal dummy connection)"
Jan 14 09:18:43 server docker[15789]: 192.168.1.143 - - [14/Jan/2018:09:18:43 -0500] "GET /apps/fulltextsearch/v1/search?request=%7B%22providers%22%3A%22files%22%2C%22search%22%3A%22tax%22%2C%22page%22%3A1%2C%22options%22%3A%7B%22files_local%22%3A%22%22%2C%22files_external%22%3A%22%22%2C%22files_extension%22%3A%22%22%7D%2C%22size%22%3A20%7D HTTP/1.1" 201 1071 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36"
Jan 14 09:18:45 server docker[15789]: 192.168.1.143 - - [14/Jan/2018:09:18:45 -0500] "GET /apps/fulltextsearch/v1/search?request=%7B%22providers%22%3A%22files%22%2C%22search%22%3A%22taxes%22%2C%22page%22%3A1%2C%22options%22%3A%7B%22files_local%22%3A%22%22%2C%22files_external%22%3A%22%22%2C%22files_extension%22%3A%22%22%7D%2C%22size%22%3A20%7D HTTP/1.1" 201 1073 "-" "Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/63.0.3239.132 Safari/537.36"
Jan 14 09:18:52 server docker[15789]: 127.0.0.1 - - [14/Jan/2018:09:18:52 -0500] "OPTIONS * HTTP/1.0" 200 126 "-" "Apache/2.4.10 (Debian) PHP/7.1.13 (internal dummy connection)"
```

In the nexcloud gui, I simply get "no results".

**Comment by bkanuka** [comment]
Ah. This has to do with files being on External storage (within nextcloud). Files under my user's directory can be indexed fine.

**Comment by daita** [comment]
Thanks for your report.

What kind of external storage are you using ?

**Comment by bkanuka** [comment]
I get the same result with local dir, SFTP (ssh kind), and FTP.

**Comment by daita** [comment]
should be fixed in 0.5.1

**Comment by daita** [comment]
re-open issue if it's not fixed.
